src/foam/nanos/auth/UserRegistrationDAO.js

```diff
@@ -0,0 +1,122 @@
+/**
+ * @license
+ * Copyright 2024 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.nanos.auth',
+  name: 'UserRegistrationDAO',
+  extends: 'foam.dao.ProxyDAO',
```

Comment:
Alternatively you could extend NullDAO to get the do-nothing impelmentation for most methods and then manually define a delegate property that you use for put().

But either way is probably fine.